lib/lightning_web/live/project_live/settings.html.heex

```diff
@@ -0,0 +1,112 @@
+<Layout.page_content>
+  <:header>
+    <Layout.header socket={@socket}>
+      <:title><%= @page_title %></:title>
+    </Layout.header>
+  </:header>
+  <Layout.centered>
+    <div class=" relative flex">
+      <div class="pr-4 pt-0 sticky top-0 self-start rounded-md">
+        <div class="flex-none">
+          <LightningWeb.Components.Common.tab_bar
+            orientation="vertical"
+            id={@project.id}
+            default_hash="settings"
+          >
+            <LightningWeb.Components.Common.tab_item
+              orientation="vertical"
+              hash="settings"
+            >
+              <Heroicons.clipboard class="h-5 w-5 inline-block mr-2 align-middle" />
+              <span class="inline-block align-middle">Project</span>
+            </LightningWeb.Components.Common.tab_item>
+            <LightningWeb.Components.Common.tab_item
+              orientation="vertical"
+              hash="credentials"
+            >
+              <Heroicons.key class="h-5 w-5 inline-block mr-2" />
+              <span class="inline-block align-middle">Credentials</span>
+            </LightningWeb.Components.Common.tab_item>
+            <LightningWeb.Components.Common.tab_item
+              orientation="vertical"
+              hash="collaboration"
+            >
+              <Heroicons.users class="h-5 w-5 inline-block mr-2" />
+              <span class="inline-block align-middle">Collaboration</span>
+            </LightningWeb.Components.Common.tab_item>
+          </LightningWeb.Components.Common.tab_bar>
+        </div>
+      </div>
+
+      <div class="flex-1 flex flex-col h-full w-full border-separate border-spacing-y-4 text-left text-sm text-gray-500 dark:text-gray-400">
+        <LightningWeb.Components.Common.panel_content for_hash="settings">
+          <div id="project-settings-#" class="bg-white p-4 rounded-md">
+            <div>
+              <h6 class="font-medium text-black">Project details</h6>
+              <small class="block my-1 text-xs text-gray-600">
+                This is how the project will appear to other collaborators
+              </small>
+            </div>
+            <.form
+              :let={f}
+              for={@changeset}
+              id="project-settings-form"
+              phx-change="validate"
+              phx-submit="save"
+            >
+              <div class="hidden sm:block" aria-hidden="true">
+                <div class="py-2"></div>
+              </div>
+              <div class="grid grid-cols gap-6">
+                <div class="md:col-span-2">
+                  <LightningWeb.Components.Form.text_field
+                    form={f}
+                    disabled={!@is_user_admin}
```

Comment:
Again for context, this should be in the form of `!@can_edit_project` or similar... basically my rough rule for these variables is to start with the word `can_...` and use that to find the verb and subject..

Reply:
Done